infrastructure/loadtesting/terraform/osquery_perf/enroll.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e
```
We should probably exit on any error rather than trying to continue.